tests/AuthenticationTest.php

```diff
@@ -46,7 +46,7 @@ public function user_gets_unauthorized_with_wrong_credentials_with_auth_middlewa
     public function user_gets_ok_with_correct_credentials_with_auth_middleware()
     {
         $user = factory(User::class)->create([
-            'password' => bcrypt('123456'),
+            'password' => password_hash('123456', PASSWORD_DEFAULT),
```

Comment:
https://laravel.com/docs/5.4/hashing#basic-usage

Reply:
@arubacao sure, expect the change on Monday.